From: Jaldhar H. Vyas <jaldhar@debian.org>
Date: Fri, 16 Sep 2011 00:55:02 +0200
Subject: default location for user mailboxes
Description: Set a default for the case where a user has no mail so auto-
 detection of mailboxes doesn't work.  See also: chgrp-error.patch
Bug: #623440
---
 doc/example-config/conf.d/10-mail.conf |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: dovecot/doc/example-config/conf.d/10-mail.conf
===================================================================
--- dovecot.orig/doc/example-config/conf.d/10-mail.conf	2013-08-19 11:23:48.532344413 -0400
+++ dovecot/doc/example-config/conf.d/10-mail.conf	2013-08-19 11:23:48.528344521 -0400
@@ -27,7 +27,7 @@
 #
 # <doc/wiki/MailLocation.txt>
 #
-#mail_location = 
+mail_location = mbox:~/mail:INBOX=/var/mail/%u
 
 # If you need to set multiple mailbox locations or want to change default
 # namespace settings, you can do it by defining namespace sections.
